addons/event/tests/test_event_internals.py

```diff
@@ -58,6 +58,27 @@ def test_event_date_computation(self):
         })
         self.assertTrue(event.is_one_day)
 
+        # Checks case when mocked today changes date before event, when event.date_tz considered
+        self.mock_datetime.now.return_value = datetime(2020, 6, 20, 20, 0, 0)
+        event.write({
+            'date_begin': datetime(2020, 6, 27, 1, 0, 0),
+            'date_end': datetime(2020, 7, 8, 2, 0, 0),
+            'date_tz': 'America/Los_Angeles'
+        })
+        self.assertEqual(registration.get_date_range_str(), u'in 6 days')
```

Comment:
Would be great to add a small explanation based on the computation for the value returned like:

```suggestion
        # date_begin_tz = 2020-06-26 18:00
        # now_tz = 2020-06-20 13:00
        # date_begin_tz.date() - now_tz.date() = 6 days
        self.assertEqual(registration.get_date_range_str(), u'in 6 days')
```

Same thing for the other case